Extension/CloudFoundry/cloudfoundry.ts

```diff
@@ -1,18 +1,15 @@
 // Copyright (c) Microsoft. All rights reserved.
 // Licensed under the MIT license. See LICENSE file in the project root for full license information.
 
-/// <reference path="../../definitions/node.d.ts"/>
-/// <reference path="../../definitions/Q.d.ts" />
-/// <reference path="../../definitions/vsts-task-lib.d.ts" />
-
 import tl = require('vsts-task-lib/task');
 import path = require('path');
 import fs = require('fs');
+import process = require('process');
 import Q = require('q');
 
 var onError = function(errMsg) {
     tl.error(errMsg);
-    tl.exit(1);
+    process.exit(1);
```

Comment:
This is not a recommended approach, since it could cause logs to not be uploaded correctly. Fixing this older pattern will require modernizing the task with async/await